raiden/blockchain_events_handler.py

```diff
@@ -321,8 +328,9 @@ def on_blockchain_event2(raiden, event):
         raise NotImplementedError('handle_channel_unlock not implemented yet')
 
     elif data['event'] == EVENT_TRANSFER_UPDATED:
-        # handle_channel_updated(raiden, event)
-        raise NotImplementedError('handle_channel_updated not implemented yet')
+        # balance proof updates are handled in the linked code, so no action is needed here
+        # https://github.com/raiden-network/raiden/blob/da54ef4b20fb006c126fcb091b18269314c2003b/raiden/transfer/channel.py#L1337-L1344  # noqa
```

Comment:
@hackaugusto , maybe there is no need to do anything at this point. However, when we integrate the MS, do you think it's good to have a check to make sure the BP is correct? If yes, maybe a comment here stating this would be nice.

Reply:
Definitely when we have the MS a check will be required, it may be the case that the node was offline and an old balance proof was provided, so the node must check the data that is provided. Currently this is not necessary because the raiden node is calling the update regardless of this smart contract state.